### PR TITLE
update jenkins credential

### PR DESCRIPTION
**Pull request by bschwedler**
- Update Jenkins credental
- Add retry(1) to several blocks in Jenkinsfile

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 
 properties([
-    disableConcurrentBuilds(),
+    disableConcurrentBuilds(abortPrevious: true),
```
Comment (kfeinauer) [inline Jenkinsfile:29]: Wasn't there an issue with enabling this? @msarahan would know?

**Review by kfeinauer — COMMENTED** [review]
Curious why the retry blocks were added. Seems like an anti pattern?

File: Jenkinsfile (hunk 1 above)
Comment (msarahan) [reply Jenkinsfile:29]: I think it should be fine. It's a multi-branch pipeline, and this implies that no more than one build can be happening per-branch, but separate branches may still run concurrently. I think that's fine, but I don't know this pipeline well.

**Comment by bschwedler** [comment]
> Curious why the retry blocks were added. Seems like an anti pattern?

We were having network failures in a few different places, particularly with the downloading of an package signing key from keyserver.ubuntu.com, which has been unstable for several people/builds.
Since the pipeline takes so long right now, it is less of a headache to have Jenkins retry specific steps.

We use this elsewhere in the Launcher builds for similar reasons.
https://github.com/search?q=repo%3Arstudio%2Flauncher+%2Fretry%5C%28%5B0-9%5D%5C%29%2F&type=code

@kfeinauer I can remove the initial commit [7b6b6a5](https://github.com/rstudio/rstudio-launcher-plugin-sdk/pull/75/commits/7b6b6a5a2c51b701bada9f2140daf40df759d01a) of retries since that didn't help the root issue, but we'll need to have at least some of them in there or we'll have to manually intervene to retry on intermittent failures.

File: Jenkinsfile (hunk 1 above)
Comment (bschwedler) [reply Jenkinsfile:29]: Since this is a multi-branch pipeline, we won't run into the same issues we saw months ago on a single branch pipeline.

**Review by kfeinauer — APPROVED** [review]

**Pull request merged**